ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -110,7 +110,7 @@ final class CommandsValidator(ledgerId: LedgerId) {
     }
   }
 
-  private def validateInnerCommands(
```

Comment:
Since motivation only in external repository, can you please make a comment that this is needed for external usage? (to prevent to move this back to private)

Reply:
Sorry, that slipped due to atuo-merge being enabled. Will create a follow-up PR.